src/main/java/com/icthh/xm/ms/entity/service/XmEntitySpecService.java

```diff
@@ -377,9 +311,9 @@ public TypeSpec findTypeByKey(String key) {
      */
     @IgnoreLogginAspect
     public Map<String, Map<String, Set<String>>> getAllKeys() {
-        Map<String, Map<String, Set<String>>> result = new HashMap<>();
+        Map<String, Map<String, Set<String>>> result = Maps.newHashMap();
```

Comment:
We in java 6?
why we try to do what recommended for java 6?
from java 7 we didn't have reason to do it!

Reply:
It is **personal preference**. Actually you are doing it Optional.of() and other static factories. Josh Block recommendation is still fine for Java. Constructor versus Factory is not 6,7,8 discussion.